xml/installation-ardana-manila.xml

```diff
@@ -70,6 +70,10 @@
 &prompt.ardana;ansible-playbook -i hosts/verb_hosts ardana-gen-hosts-file.yml
 &prompt.ardana;ansible-playbook -i hosts/verb_hosts FND-CLU-reconfigure.yml
 &prompt.ardana;ansible-playbook -i hosts/verb_hosts ardana-reconfigure.yml</screen>
+    If &o_sharefs has already been installed but is being reconfigured run the following for
```

Comment:
`&o_sharefs` needs to be `&o_sharefs;` I think

Reply:
done